app/src/main/java/io/intrepid/contest/screens/contestcreation/editcategoriestocontest/EditCategoriesFragment.java

```diff
@@ -23,6 +25,14 @@
     EditText categoryDescriptionField;
     private ActivityCallback activity;
 
+    public static Fragment newInstance(@Nullable Category category) {
+        EditCategoriesFragment fragment = new EditCategoriesFragment();
+        Bundle args = new Bundle();
+        args.putParcelable(CATEGORY_KEY, category);
+        fragment.setArguments(args);
```

Comment:
Yeah It looks a little weird at first because it almost looks like the bundled arguments are meant to go to some other class or activity. but you can think of it as basically the same as an intent extra. It's kinda similar to the static intent pattern we've been following but for the fragment